charts/cloudzero-agent/values.yaml

```diff
@@ -93,6 +92,10 @@ server:
   - --web.console.libraries=/etc/prometheus/console_libraries
   - --web.console.templates=/etc/prometheus/consoles
   - --enable-feature=agent
+  # -- The agent will use this file path on the container filesystem to get the CZ API key.
+  containerSecretFilePath: /etc/config/prometheus/secrets/
+  # -- The agent will look for a file with this name to get the CZ API key.
+  containerSecretFileName: value
```

Comment:
I originally named with `cz-api-key`, which I think is a better name than `value`. but I'm worried that users that have created secrets externally with the `value` name could see this fail on upgrade